consensus/rewards.md

```diff
@@ -0,0 +1,35 @@
+# Miners' rewards
+
+Aeternity is Proof of Work blockchain. This is a description of rewards paid to miners.
+
+## Introduction
+
+Miners mine key blocks and receive rewards composed out of two ingredients:
+1. Coinbase-like reward for mining a key block
+2. Fees and gas paid by users for transactions and contract calls
+
+There is also special case altering rewarding scheme. It crypto economic incentive designed to lower risk of fraud - and it is called Proof of Fraud.
```

Comment:
This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L86).